design/stackhub.md

```diff
@@ -0,0 +1,207 @@
+# Collection transition to Stack Design
+
+## Key Concepts / Background
+- Simplify key concepts by eliminating Collections and Collection Hubs
+  -  Object model inheritence, e.g. A Collection ISA a Stack and a CollectionHub ISA a Stackhub, causes confusion especially for community messsaging, developers and new comers to the project.
+  -  Monolithic build (CollectionHub) causes painful lifecycle for making configuration or manage the curated stacks and pipelines.
+
+- In general, to replace Collection Hubs with Stack Hubs, and providing configuration for the governed stacks via the defined Kubernetes Custom Resources:  Kabanero CR.
+
+- Renaming the collection CR a stack CR, which has essentially the same data and format.
+
+- Although this is not a seemingly drastic change -- the concept of CollectionHub and Collections has tentacles throughout the website, guides, documentation, videos and blogs.
+
+## User stories
+
+- As the Kabanero community, we would like to simplify the messaging of Kabanero by removing the concept of CollectionHub and Collections.
+
+- As Champ (architect), I would like the definition, selection and build of Appsody stacks that are "blessed" untangled from the configuration of a specific Kabanero instance, moving from the Collection-Hub concept to a strictly Kubernetes-natural Kabanero and Stack CRD gitops user experience.
```

Comment:
I am of the opinion that we should try to avoid multiple points of configuration. I would think the user experience would be much simpler if there is only one configuration for them to be concerned with.